Makefile

```diff
@@ -3,13 +3,32 @@
 # These variables can be overridden by setting an environment variable.
 LOCAL_IP_ENV?=$(shell ip route get 8.8.8.8 | head -1 | cut -d' ' -f8)
 
+CONFD_REPO?=calico/confd
+CONFD_VER?=latest
+CONFD_CONTAINER_NAME=${CONFD_REPO}:${CONFD_VER}
+
 default: clean calicorr.created
 
-calicorr.created: $(BUILD_FILES)
+calicorr.created: $(BUILD_FILES) dist/confd
 	docker build -t calico/routereflector .
 	touch calicorr.created
 
 clean:
 	-rm *.created
-	-docker rm -f calico-rr
-	-docker rmi -f calico/rr
+	-rm -rf dist
+
+dist/confd: dist
+	-docker rm -f calico-confd
+	# Latest confd binaries are stored in automated builds of calico/confd.
+	# To get them, we create (but don't start) a container from that image.
+	#docker pull $(CONFD_CONTAINER_NAME)
```

Comment:
should we re-instate this docker pull?

Reply:
Yes, and I've done that now, at the same time as referencing a specific new calico/confd version.